tests/views/flash/nested_checkbox.blade.php

```diff
@@ -0,0 +1,3 @@
+{{ \Form::with(null, ['url'=>'/test-other-route']) }}
+  {{ \Form::checkBoxWithObject('post[some][nested][path]', 'is_published', $options) }}
+{{ \Form::end() }}
```

Comment:
This is not nested from Forms point of view, but not a bad test by itself.

Add a test with `Form::fieldsFor` nesting